docs/tutorials/1_the_ibmq_account.ipynb

```diff
@@ -23,13 +23,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In Qiskit we have an interface for backends and jobs that is useful for running circuits and extending to third-party backends. In this tutorial, we will review the core components of Qiskit’s base backend framework, using the IBM Quantum Experience (IQX) account as an example.\n",
+    "In Qiskit we have an interface for backends and jobs that is useful for running circuits and extending to third-party backends. In this tutorial, we will review the core components of the base backend framework of Qiskit, using the IBM Quantum Experience (IQX) account as an example.\n",
     "\n",
     "The interface has four main components: the account, providers, backends, and jobs:\n",
     "\n",
-    "- **account**: Gives access to one or more ‘providers’ based on the account's permissions.\n",
+    "- **account**: Gives access to one or more 'providers' based on the account's permissions.\n",
     "\n",
-    "- **provider**: Provides access to quantum devices and simulators, collectively called ‘backends’, and additional services tailored to a specific backend instance.\n",
+    "- **provider**: Provides access to quantum devices and simulators, collectively called ‘backends’ and additional services tailored to a specific backend instance.\n",
```

Comment:
```suggestion
    "- **provider**: Provides access to quantum devices and simulators, collectively called 'backends' and additional services tailored to a specific backend instance.\n",
```
This is a nit, but since the backtick on line 30 was changed, this should match that.